src/adapters/github.ts

```diff
@@ -56,9 +58,13 @@ class GithubAdapter extends GitAdapter {
   }
 
   public async getCandidateRepos(onRetry: RetryMethod): Promise<IRepo[]> {
-    const { org, search_query } = this.migrationContext.migration.spec.adapter;
+    const { org, search_type, search_query } = this.migrationContext.migration.spec.adapter;
     let repoNames = [];
 
+    if (search_type && !GithubAdapter.validSearchTypes.has(search_type)) {
+      throw new Error(`Parameter "search_type" must be one of the following: ${Array.from(GithubAdapter.validSearchTypes).map(e => `'${e}'`).join(', ')}`);
```

Comment:
Nit: we don't really refer to these as parameters elsewhere - let's drop that from the error message.

```suggestion
      throw new Error(`"search_type" must be one of the following: ${Array.from(GithubAdapter.validSearchTypes).map(e => `'${e}'`).join(', ')}`);
```